site/src/pages/release-notes/1.20.0.mdx

```diff
@@ -0,0 +1,150 @@
+---
+date: 2022-10-05
+---
+
+## 🌟 New features
+
+- You can now convert the GraphQL Error into an <type://HttpResponse> using <type://GraphqlErrorHandler>. #4435
```

Comment:
As it is not just a conversion, I slightly prefer to use `handle` rather than `covert`.
```suggestion
- You can now handle the GraphQL Error using <type://GraphqlErrorHandler>. #4435
```